src/components/hamburgerMenu.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+import "./hamburgerMenu.css"
+
+export default function HamburgerMenu({
+  headerRef,
+  setMobileExpanded,
+  mobileNavRef,
+}) {
+  const [active, setActive] = useState(false)
+  const inactiveClass = "hamburger hamburger--slider"
+  const activeClass = "hamburger hamburger--slider is-active"
+
+  const handleClick = () => {
+    setActive(!active)
+
+    if (!active) {
+      headerRef.current.classList.add("header--secondary")
+      headerRef.current.classList.remove("header--primary")
+      mobileNavRef.current.style.height = "500px"
```

Comment:
I think we can achieve this by targeting a class name and using breakpoints instead. That will probably help with the issue when resizing back and forth between mobile and desktop.